resources/views/auth/login.blade.php

```diff
@@ -18,12 +18,12 @@
 
                     <div class="form-item -padded">
                         <label class="field-label" for="email">Email</label>
-                        <input type="text" name="email" id="email" class="text-field" placeholder="kallark&#64;dosomething.org" required>
+                        <input type="text" name="email" id="email" class="text-field" placeholder="kallark&#64;dosomething.org">
```

Comment:
lol. nope, so I can show nice customized validation errors with our own Forge interface instead of the default HTML5 popup :)